### PR TITLE
Event model

### PR DESCRIPTION
**Pull request by lpile**
## Changes proposed in this pull request:
* Setup RSpec, ShouldaMatcher, FactoryBot, Faker, and SimpleCov gems
* Add Event model with spec
* Add request event index spec
* Create #index action for EventsController with route

## Card(s) closed in this pull request:
* Doesn't close any cards at the moment.

## What did you struggle on to complete?
* Anneke seeing Rails for the first time.

### Test Coverage Percentage: 100%
- [x] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

## Helpful Resources:
[Use BE mod 3 API tutorial](https://backend.turing.io/module3/lessons/exercises/building_an_api)

### CODE REVIEW

File: app/controllers/api/v1/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Api::V1::EventsController < ApplicationController
+  def index
+    render json: Event.all
```
Comment (annekemcgrady) [inline app/controllers/api/v1/events_controller.rb]: Logan, will you please add comments as you are writing to help me when I'm following along on my own. Thanks!

Comment (lpile) [reply app/controllers/api/v1/events_controller.rb]: Of course, I will start doing that now.
